### PR TITLE
New Run in Postman button

### PR DESCRIPTION
**Pull request by sowmyaraj92**

### CODE REVIEW

File: src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md [hunk 1]
```diff
@@ -40,47 +40,49 @@ contextual_links:
 
 ---
 
-One way to [share your Postman Collections](/docs/collaborating-in-postman/sharing/) is to create a stand-alone Run in Postman button. You can embed the button in your website or a README to let developers interact with your API more quickly.
+One way to [share your Postman Collections](/docs/collaborating-in-postman/sharing/) is to create a stand-alone Run in Postman button. You can embed the button in your website or a README to let developers interact with your API more quickly. The new Run in Postman button allows you to fork the collections so that they stay up to date by pulling and merging changes.
```
Comment (jkonrath-postman) [inline src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md:43]: General question: should be we bolding the **Run in Postman** button like this?

File: src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md [hunk 2]
```diff
@@ -40,47 +40,49 @@ contextual_links:
...
-3. Tab over to **Embed**.
-4. Click **Generate code**.
-5. Choose a dynamic or static button:
-   * **Dynamic button**: This button uses JavaScript, HTML and CSS, so you can customize the button for a website. You can also access the [Run in Postman API](/docs/publishing-your-api/run-in-postman/run-button-API/) to dynamically create and update environments.
-   * **Static button**: This button uses Markdown, so you can display it in a README, blog, or other Markdown document.
-6. You can optionally include an environment to embed with your collection. On the right, click the **Environment** drop-down and choose the environment.
-7. Click **Copy to clipboard**.
-8. Embed the code where you would like the button to be displayed.
+> Note: If the collection is present in a public workpspace, you can directly embed the copied code where you would like the button to be displayed. If the collection is present in a team or a personal workspace, [share collection to a public workspace](/docs/collaborating-in-postman/using-workspaces/managing-workspaces/#adding-elements-to-workspaces) to create the run in postman button.
```
Comment (jkonrath-postman) [inline src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md:56]: * Change "share collection to a public workplace" to "share the collection to a public workspace"
* Change "run in postman button" to "Run in Postman button" (and maybe bold, see the previous comment)

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
Some minor comments, but make sure to search/replace everything to make sure it's Run in Postman and maybe always make the button name bold?

File: src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md [hunk 3]
```diff
@@ -40,47 +40,49 @@ contextual_links:
...
+One way to [share your Postman Collections](/docs/collaborating-in-postman/sharing/) is to create a stand-alone Run in Postman button. You can embed the button in your website or a README to let developers interact with your API more quickly. The new **Run in Postman** button allows you to fork the collections so that they stay up to date by pulling and merging changes.
 
-> The Run in Postman button also works with API specification formats like OpenAPI and RAML. To create a button for the standard you use, first convert the file to a collection by [importing it into Postman](/docs/getting-started/importing-and-exporting-data/) or by [authoring your specification in Postman](/docs/designing-and-developing-your-api/the-api-workflow/) to generate a collection.
+> Previously, the Run in Postman button worked with API specification formats like OpenAPI and RAML. To create a button for the standard you use, first convert the file to a collection by [importing it into Postman](/docs/getting-started/importing-and-exporting-data/) or by [authoring your specification in Postman](/docs/designing-and-developing-your-api/the-api-workflow/) to generate a collection.
```
Comment (gitThingsDone) [inline src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md:45]: users can still create these run in postman buttons from API specification formats

**Review by gitThingsDone — CHANGES_REQUESTED** [review]
Added one comment, looks good to go otherwise :D

**Review by gitThingsDone — APPROVED** [review]

File: src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md [hunk 4]
```diff
@@ -40,47 +40,50 @@ contextual_links:
...
+One way to [share your Postman Collections](/docs/collaborating-in-postman/sharing/) is to create a stand-alone Run in Postman button.  The new **Run in Postman** button allows you to fork the collections. You can embed the button in your website or a README to let developers interact with your API more quickly.
 
-> The Run in Postman button also works with API specification formats like OpenAPI and RAML. To create a button for the standard you use, first convert the file to a collection by [importing it into Postman](/docs/getting-started/importing-and-exporting-data/) or by [authoring your specification in Postman](/docs/designing-and-developing-your-api/the-api-workflow/) to generate a collection.
+> Being a user, you can still create **Run in Postman** button from API specification formats like OpenAPI and RAML. To create a button for the standard you use, first convert the file to a collection by [importing it into Postman](/docs/getting-started/importing-and-exporting-data/) or by [authoring your specification in Postman](/docs/designing-and-developing-your-api/the-api-workflow/) to generate a collection.
```
Comment (jkonrath-postman) [inline src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md:45]: "can still create a" or "can still create the" ?

File: src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md [hunk 5]
```diff
@@ -92,19 +95,17 @@ data-postman-var-1=":collection_id"></div>
...
+> You can include an environment along with a collection in your Run in Postman button. The dynamic button gives you access to the [Run in Postman API](/docs/publishing-your-api/run-in-postman/run-button-API/) to create and modify environments.
 
-1. In the Postman app, open the original collection.
-2. Next to the collection name, expand the collection details and click the **Share** button to open a modal.
-3. Tab over to **Embed** > **Update link**.
+Clicking **Run in postman** button navigates to the page where you can fork the collection to your workspace. [Forking the collection](/docs/collaborating-in-postman/version-control-for-collections/) into your workspace will enable you to contribute to the source collection using pull requests. You can also view the collection in a public workspace if you like and even import a copy of the collection using the links present on the screen. All collections shared with the new Run in postman buttons come with [Fork counts](/docs/collaborating-in-postman/version-control-for-collections/#forking-information), that help you and your consumers understand how the API is being used.
```
Comment (jkonrath-postman) [inline src/pages/docs/publishing-your-api/run-in-postman/creating-run-button.md:104]: Should be "Run in Postman"

**Review by jkonrath-postman — APPROVED** [review]
Approved, but there are two super-minor changes for you to pick up if you can.

**Pull request merged**